src/Zicht/Bundle/FrameworkExtraBundle/Command/ValidateEntityCommand.php

```diff
@@ -68,7 +59,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
     /**
      * get all entities
      *
-     * @return \Zicht\Itertools\lib\FilterIterator
+     * @return \Generator
```

Comment:
This change occurs often in this PR, that is fine, but it does introduce a breaking change, as the `FilterIterator` provides many more options than a `Generator` (i.e. `->first()`, etc).

So this PR should become a new major version.

Also, an entry in the changelog file is required.